modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc

```diff
@@ -0,0 +1,22 @@
+= Zuora - XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== XML Namespace
+
+<No information>
+
+== POM File Configuration
+
+To use this connector with Maven, view the pom.xml dependency information in the Dependency Snippets in Anypoint Exchange.
+
+== Keep Session Alive
```

Comment:
Ok to remove this section -- not in the template

Reply:
Do you mean the Keep Session Alive section?